dkms.in

```diff
@@ -2181,6 +2184,11 @@ autoinstall() {
         IFS=/ read m v <<< "$mv"
         echo "$m/$v autoinstall failed due to missing dependencies: ${build_depends[$m]}"
     done
+
+    if [[ "${#failed_modules[@]}" > 0 || "${#to_install[@]}" > 0 ]]; then
```

Comment:
Note to self (et al): should really be using `(( variable > 0 || another > 0 ))`, as mentioned in https://github.com/dell/dkms/issues/268

Reply:
Is that something you want fixed in this submission? Or leave it for whoever gets around to that issue?

Reply:
It's fine to leave for later IMHO.